bancorprotocol/carbon-app
# Strategy Table List

fix #1585 

Test case : 
- Go to explorer
- Click on the table icon next to the sort/filter
- Reload -> should still have table
- Open a new tab and go to explorer -> should be table by default now
